android/src/main/java/io/textile/rnmobile/IpfsBridge.java

```diff
@@ -39,6 +39,25 @@ public void run() {
         });
     }
 
+    @ReactMethod
+    public void connect(final String multiaddr, final Promise promise) {
```

Comment:
Can we just update this to `swarmConnect` everywhere for consistency with the native sdks?

Reply:
I just named it for consistency with the js-http-client sdk , ref to https://github.com/textileio/js-http-client/blob/5f8b3a421ca682b3ba5ed3d4af96bca1d0913a39/src/modules/ipfs.ts#L58 ... So should we still update this to swarmConnect?

Reply:
Ah ok, that makes sense. Let's just leave this as is then.